src/api/test/test_helper.rb

```diff
@@ -138,7 +140,7 @@ class ActionDispatch::IntegrationTest
   # but for debugging or for deep testing the check can be run after each test case
   def after_teardown
     super
-    if false
+    if DEBUG_MODE
```

Comment:
Commenting the code? We should need to change the variable to true to execute this anyway, so uncommenting it would be the same. :thinking: